src/components/ui/NavItem.tsx

```diff
@@ -0,0 +1,13 @@
+interface NavItemProps {
```

Comment:
Is this component being used?

Reply:
I was unable to adapt all my components into smaller components, and in order to keep the same structure and layout I decided not to use the ones I created and just keep the initial code structure

Reply:
I see, for future works try to not send incomplete stuff